src/v2/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarCommercial.jest.tsx

```diff
@@ -151,6 +154,47 @@ describe("ArtworkSidebarCommercial", () => {
     expect(wrapper.text()).toContain("Make offer")
   })
 
+  it("displays both Make Offer and Contact Gallery CTAs when offerable from inquiry and exact price listed", () => {
```

Comment:
Please don't create any new tests with Enzyme, we're migrating bit by bit to RTL which is reliable and more efficient to the tests, if it is too much refactoring the test file at least the newer ones use RTL, see more on [link](https://github.com/artsy/force/blob/main/docs/best_practices.md#write-unit-tests-for-new-components)